crates/notion-core/src/inventory/mod.rs

```diff
@@ -567,17 +571,28 @@ pub struct NodeDistroFiles {
 
 /// Reads a public index from the Node cache, if it exists and hasn't expired.
 fn read_cached_opt() -> Fallible<Option<serial::NodeIndex>> {
-    let expiry: Option<String> = read_file_opt(&path::node_index_expiry_file()?).unknown()?;
+    let expiry_file = path::node_index_expiry_file()?;
+    let expiry: Option<String> =
```

Comment:
This type annotation is unnecessary, so we can drop it while we're in here.

Reply:
Fixed